client/python/lib/README.md

```diff
@@ -3,37 +3,32 @@
 Model server client library is a set of objects and methods designed to simplify user interaction with the instance of the model server. The library contains functions that hide API specific aspects so user doesn't have to know about creating protos, preparing requests, reading responses etc. and can focus on the application itself rather than dealing with API calls to OVMS.
 
 
-## Building pip package
+## Installation
 
-The library is distributed in form of pip package. To build and install the package first clone model server repository:
+The client library requires Python in version >= 3.6. Validated and recommended way of installing the library is with the use of virtual python environment.
 
-`git clone https://github.com/openvinotoolkit/model_server.git`
+### Linux
+For one-step building and installation simply run:
+`make install` 
```

Comment:
perhaps it would be easier just to generate wheel package and from that point anyone could install it in any environment?
Do we need to require make command to install the client?

Reply:
This make has been created to make one-step installation possible. I didn't want to split the installation to something like:
```
make build-package
pip install dist/ovmsclient-0.1-py3-none-any.whl
```
Do you think it's better to remove installation step from it?